images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -30,88 +30,91 @@ if (Test-IsWin19)
 $markdown += New-MDHeader "Installed Software" -Level 2
 $markdown += New-MDHeader "Language and Runtime" -Level 3
 
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-GoVersion)
+)
+
 $markdown += New-MDList -Lines (Get-JavaVersionsList -DefaultVersion "1.8.0") -Style Unordered -NoNewLine
 $markdown += New-MDList -Style Unordered -Lines @(
     (Get-BashVersion),
-    (Get-PythonVersion),
-    (Get-RubyVersion),
-    (Get-GoVersion),
-    (Get-PHPVersion),
     (Get-JuliaVersion),
+    (Get-NodeVersion),
     (Get-PerlVersion),
-    (Get-NodeVersion)
+    (Get-PHPVersion),
+    (Get-PythonVersion),
+    (Get-RubyVersion)   
 )
 
 $markdown += New-MDHeader "Package Management" -Level 3
 $markdown += New-MDList -Style Unordered -Lines @(
     (Get-ChocoVersion),
-    (Get-VcpkgVersion),
+    (Get-ComposerVersion),
+    (Get-HelmVersion),
+    (Get-CondaVersion),
     (Get-NPMVersion),
-    (Get-YarnVersion),
+    (Get-NugetVersion),
+    (Get-PipxVersion),
     (Get-PipVersion),
-    (Get-CondaVersion),
     (Get-RubyGemsVersion),
-    (Get-HelmVersion),
-    (Get-ComposerVersion),
-    (Get-NugetVersion),
-    (Get-PipxVersion)
+    (Get-VcpkgVersion),
+    (Get-YarnVersion)
 )
 
 $markdown += New-MDHeader "Project Management" -Level 3
 $markdown += New-MDList -Style Unordered -Lines @(
```

Comment:
May be we should add ` -Lines (@(...) | Sort-Object)` to every array to be sure that they are sorted properly?

Reply:
Thank you, added `Sort-Object`